FhirDeathRecord/DeathRecord.cs

```diff
@@ -45,11 +47,12 @@ private string GetFirst(string path)
             var matches = Navigator.Select(path);
             if (matches.Count() > 0)
             {
-                return (string)matches.First().Value;
+                return Convert.ToString(matches.First().Value);
             }
             else
             {
                 // TODO: Figure out appropriate way to handle element not found
+                Console.WriteLine("element not found");

```

Comment:
I've thought a bit more about this case. I think (and am doing this locally right now with my ongoing additions) we should return `null` here. My thoughts are:
- It seems easier, or perhaps more clear, to check for `null` inside any app that wants to use this code, when checking to see if a "thing" exists in a record. We shouldn't be doing anything special, or making **any** assumptions on behalf of any users of this library.
- We shouldn't be printing anything to the console.
- In our CLI app, we print whatever these functions return. The output is the same for `null` and `""` (Console.WriteLine outputs nothing for both).

For example, `DeathRecord.AutopsyPerformed()` returning `null` if the record does not have an entry for Autopsy Performed seems the most natural thing to me.

Pinging @pkmitre for his thoughts on this as well, as the TODO was his original comment.

Reply:
I tried to think a bit about what it would look like if we raised an error; that doesn't seem right, since we couldn't really bubble it up to the user because the user needs to be able to ignore missing fields. `null` seems reasonable to me; one reason not to do that would be if the (IJE-inspired) receiving interface expects a string, but that seems likely easy to deal with.